app/templates/components/ui-table/cell/events/view/sessions/cell-session-title.hbs

```diff
@@ -0,0 +1,15 @@
+{{record.title}}
```

Comment:
Also there's no point of using a property name if you are using record.title simply to get the title.
May be you can use (get record column.propertyName) for it.